Formula/kore.rb

```diff
@@ -16,8 +16,15 @@ class Kore < Formula
   # Reported 4 Aug 2016: https://github.com/jorisvink/kore/issues/140
   depends_on :macos => :yosemite
 
+  # dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
+  # Reported 20 Aug 2017 to graeme AT argyllcms DOT com
+  if MacOS.version == :el_capitan && MacOS::Xcode.installed? && MacOS::Xcode.version >= "8.0"
+    inreplace "numlib/numsup.c", "CLOCK_MONOTONIC", "UNDEFINED_GIBBERISH"
```

Comment:
This blob will need a little bit of tweaking from the example I provided. In this case the file should change from `numlib/numsup.c` to `src/utils.c`, and the comments from:

```
# dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
# Reported 20 Aug 2017 to graeme AT argyllcms DOT com
```
to:
```
# error: use of undeclared identifier 'CLOCK_MONOTONIC'
# Reported <date> to <upstream bug link>
```

Reply:
Thanks for the help, I really have no idea, what this is all about. Are there any references on getting some general information about what‘s the problem?

Reply:
I can't provide an upstream bug link because the project is no longer on GitHub and is now maintained by a mailing list.

Maybe @jorisvink has some thoughts on this?

Reply:
I talked to the developer. No work around will happen just because of El Capitan. My suggestion would be to set depend_on :macos to :sierra. Would that be sufficient?

Reply:
Aye, if they're never gonna fix it we'll have to go down that route. I don't want to carry around a new `inreplace` forevermore.

```
depends_on :macos => :sierra # needs clock_gettime
```